strax/context.py

```diff
@@ -1115,6 +1117,7 @@ def get_array(self, run_id: ty.Union[str, tuple, list],
                 targets,
                 save=save,
                 max_workers=max_workers,
+                allow_multiple=False,
```

Comment:
If this argument isn't allowed in `get_array` or `get_df`, why is it here?

Reply:
to make sure that it is not passed on as a `**kwarg`

Reply:
Hm okay I see the problem, but I would prefer if we add in the first line of get_array as `kwargs['allow_multiple'] = False` to avoid error messages like "allow_multiple has been specified twice" which would for sure lead to confusion.

Reply:
This is a valid point, I changed it.